File: Q2-Assessment/ass.kt
fun main() {
product = Product("laptop",20000.00,5)
   println( product.calculateAverage())
    student = Student("ANgela",24,67.80)
   println( student.calculateAverage())
}
class Product(val name:String,val price:Double,val quantity:Int){
    fun calculateAverage():Double{
        var price = 2000
        var quantity = 5
        return price* quantity
    }
}

class Student(val name:String,val age:Int,val grades:Double){
    def calculateAverage():Double{
        var total = sum(grades)
        var avg = total/count(grades)
        return avg

    }
}
class FlightBooking(val destination:String,val date:String,val customer:Int,val flights:String){
    fun searchFlight(){
       var flightDestination = []
        var flightDate = []
        var availableFlight = mutableListOf<String>(){
            for (flight in flights){
                if (flightDestination == destination && flightDate){
                    return availableFlight
                }
            }
        }
    }
}

File: Q2-Assessment/ass.js
class Recipe{
    constructor(food, ingredients, preparationTime, cookingMethod, nutritionalInfo) {
        this.food = food;
        this.ingredients = ingredients;
        this.preparationTime = preparationTime;
        this.cookingMethod = cookingMethod;
        this.nutritionalInfo = nutritionalInfo;
    }

}

class Product {
    constructor(name, price, quantity) {
      this.name = name;
      this.price = price;
      this.quantity = quantity;
    }
    calculateTotal() {
      return this.price * this.quantity;
    }
  }
  let product1 = new Product("Laptop", 250000.99, 2);
  let  product2 = new Product("mouse", 500.00, 2);

  let totalValue1 = product1.calculateTotal();
  let totalValue2 = product2.calculateTotal();
  console.log(` ${product1.name}: $${totalValue1}`);
  console.log(` ${product2.name}: $${totalValue2}`);

  class Student{
    constructor(name,age,grades){
        this.name=name
        this.age=age
        this.grades=grades
    }
  }